### PR TITLE
max upload content size handling

### PR DESCRIPTION
**Pull request by vepkenez**

### CODE REVIEW

File: test/debug_test_application.py [hunk 1]
```diff
@@ -13,6 +13,6 @@
 if __name__ == "__main__":
     threadPool = reactor.getThreadPool()
     threadPool.adjustPoolsize(3, 5)
-    options = {'wsgi': application}
+    options = {'wsgi': application, 'max_upload_bytes': 1024}
```
Comment (KPrasch) [inline test/debug_test_application.py]: This is the default?

File: test/test_request_behavior.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+@pytest_twisted.inlineCallbacks
+def test_max_upload_bytes():
+    options = {'wsgi': application, 'max_upload_bytes': 200, 'http_port': 9876}
+    deployer = HendrixDeploy(options=options)
+    deployer.addServices()
+    deployer.start()
+
+    def reject_large_uploads():
+
+        # upload 50 bytes, we should be fine.
```
Comment (KPrasch) [inline test/test_request_behavior.py]: What's with the speculation?

```suggestion
        # uploading 50 bytes is fine.
```

File: hendrix/__init__.py [hunk 3]
```diff
@@ -1 +1 @@
-__version__ = '3.2.5'
+__version__ = '3.3.0'
```
Comment (KPrasch) [inline hendrix/__init__.py]: Did you use bumpversion for this?

Comment (vepkenez) [reply hendrix/__init__.py]: I don’t know what that means

File: test/debug_test_application.py (hunk 1 above)
Comment (vepkenez) [reply test/debug_test_application.py]: Default is unlimited to maintain compatibility

**Pull request merged**